src/components/InputNumber/InputNumber.module.css

```diff
@@ -107,6 +105,21 @@
   @apply px-6 py-3 text-base;
 }
 
+.sbui-inputnumber-nav--tiny {
+  height: 30px;
```

Comment:
if these could be done with full height, or fill the height somehow, that would be amazing.

This is incase the sizes of the inputs change in future and these heights would need to also be updated.

Reply:
don't worry though, we can just amend this later 💪

Reply:
I try to play with the idea a bit but I guess we need to manually update the height if the size of the inputs ever changes.